<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>DEDZ Counter</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="sB_D3DZC0unter">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pB_SwitchMovementMethod">
       <property name="text">
        <string>Switch Movement Method</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_Reset">
       <property name="text">
        <string>RESET Transform</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_BoundBoxRectToggle">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Show / Hide Sprite Bounding Box Rect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_Shear">
       <property name="toolTip">
        <string>To see what shear does</string>
       </property>
       <property name="text">
        <string>Test Shear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="2" rowspan="2">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTextBrowser" name="textBrowser_2"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
